Sources/Orbit/Components/InputField.swift

```diff
@@ -337,6 +234,16 @@ public extension AccessibilityID {
     static let inputFieldPasswordToggle     = Self(rawValue: "orbit.inputfield.password.toggle")
 }
 
+// MARK: - Private
+private extension InputField {
```

Comment:
These are specifically `private` in order for the autocomplete to only offer sensible builder inits. It would not matter for a call site, but it helps even in Orbit previews to get the useful methods show up first. I would consider doing that for other components later as well. (although it is mostly needed to those that would get their init builder modifiers)